Don't redefine in6_addr due to including linux/in6.h; use netinet/in.h instead.

--- a/include/linux/if_bridge.h	2015-09-01 01:31:15.000000000 +0200
+++ b/include/linux/if_bridge.h	2015-09-14 16:23:24.472434012 +0200
@@ -15,7 +15,6 @@
 
 #include <linux/types.h>
 #include <linux/if_ether.h>
-#include <linux/in6.h>
 
 #define SYSFS_BRIDGE_ATTR	"bridge"
 #define SYSFS_BRIDGE_FDB	"brforward"
--- a/include/linux/netfilter.h	2015-09-01 01:31:15.000000000 +0200
+++ b/include/linux/netfilter.h	2015-09-14 16:27:34.928205553 +0200
@@ -4,8 +4,6 @@
 #include <linux/types.h>
 
 #include <linux/sysctl.h>
-#include <linux/in.h>
-#include <linux/in6.h>
 
 /* Responses from hook functions. */
 #define NF_DROP 0
--- a/include/linux/xfrm.h	2015-09-01 01:31:15.000000000 +0200
+++ b/include/linux/xfrm.h	2015-09-14 16:23:24.473434008 +0200
@@ -1,7 +1,6 @@
 #ifndef _LINUX_XFRM_H
 #define _LINUX_XFRM_H
 
-#include <linux/in6.h>
 #include <linux/types.h>
 
 /* All of the structures in this file may not change size as they are
--- a/include/utils.h	2015-09-01 01:31:15.000000000 +0200
+++ b/include/utils.h	2015-09-14 16:23:24.472434012 +0200
@@ -1,6 +1,7 @@
 #ifndef __UTILS_H__
 #define __UTILS_H__ 1
 
+#include <sys/param.h>	/* MAXPATHLEN */
 #include <sys/types.h>
 #include <asm/types.h>
 #include <resolv.h>
--- a/include/linux/netfilter_ipv4/ip_tables.h
+++ b/include/linux/netfilter_ipv4/ip_tables.h
@@ -17,7 +17,6 @@
 
 #include <linux/types.h>
 
-#include <linux/if.h>
 #include <linux/netfilter_ipv4.h>
 
 #include <linux/netfilter/x_tables.h>
--- a/include/bpf_scm.h
+++ b/include/bpf_scm.h
@@ -3,6 +3,7 @@
 
 #include <sys/types.h>
 #include <sys/socket.h>
+#include <sys/un.h>
 
 #include "utils.h"
 #include "bpf_elf.h"
--- a/include/linux/if_tunnel.h
+++ b/include/linux/if_tunnel.h
@@ -2,9 +2,7 @@
 #define _IF_TUNNEL_H_
 
 #include <linux/types.h>
-#include <linux/if.h>
 #include <linux/ip.h>
-#include <linux/in6.h>
 #include <asm/byteorder.h>
 
 
--- a/include/linux/l2tp.h
+++ b/include/linux/l2tp.h
@@ -9,8 +9,6 @@
 
 #include <linux/types.h>
 #include <linux/socket.h>
-#include <linux/in.h>
-#include <linux/in6.h>
 
 #define IPPROTO_L2TP		115
 
